### ISSUE TITLE
xticks on inset can overlap

### ISSUE THREAD

**Issue by vext01** [softdevteam/warmup_experiment#367]
```
${EXPERIMENT_REPO}/bin/plot_krun_results --inset-xlimits 10,100 --export-size 4.5,4 --no-zoom --with-changepoint-means --with-outliers --core-cycles 3 -o examples/new_warmup_no_migrate.pdf -b bencher5:nbody:HHVM:default-php:9 ${BENCHER5_DATA}
```

![over](https://cloud.githubusercontent.com/assets/604955/24351915/851d0f64-12df-11e7-9da2-ec230765f8f2.png)

**Comment by vext01** [comment]
How about only labeling the first and last tick for insets?

**Comment by snim2** [comment]
Is this because you have a tiny `--export-size`?

**Comment by vext01** [comment]
Depends on #368 to avoid confusion.

**Comment by vext01** [comment]
Yes, probably, but I wouldn't like to make the inset larger, or decrease the font size, would you?

**Comment by snim2** [comment]
No, but you could make the export size larger.

**Comment by vext01** [comment]
Well, in the context of a fixed width latex box, which we are putting the plot in, a larger export size has the effect of making the fonts smaller...

**Comment by snim2** [comment]
Remember you are controlling the aspect ratio here. If I try something like this on the #368 branch `--export-size 4.5,4` still looks problematic, `--export-size 5, 4` looks much better.

**Comment by vext01** [comment]
Right, sometimes you get lucky, but imagine the range had been in triple or quadruple figures, you'd probably never win, right? Example above has only double figures.

**Comment by snim2** [comment]
Fixed in #368
